### ISSUE TITLE
Add standby mode for log restores to sp_DatabaseRestore

### ISSUE THREAD

**Issue by James-DBA-Anderson** [BrentOzarULTD/SQL-Server-First-Responder-Kit#1152]
Feature Request

It's not currently possible to restore the logs in standby mode. I'm not sure if this is something you want to add to this script, but I have written some code that adds it.

My code introduces two new params @StandbyMode and @StandbyUndoPath. It then decides which option to add to the log restore command with the code below:

```
IF (@StandbyMode = 1)
	BEGIN
		IF (@StandbyUndoPath IS NULL)
			RAISERROR('The file path of the undo file for standby mode was not specified. Logs will not be restored in standby mode.', 0, 1) WITH NOWAIT;
		ELSE
			SET @LogRecoveryOption = N'STANDBY = ''' + @StandbyUndoPath + @Database + 'Undo.ldf''';
	END;

IF (@LogRecoveryOption = N'')
	BEGIN
		SET @LogRecoveryOption = N'NORECOVERY';
	END;
```

If this looks like something you'd want to add then let me know and I'll create a pull request.

**Comment by BlitzErik** [comment]
Howdy!

Yeah, this looks great. We'd love it. Sorry about the late response, been a crazy month!

Look forward to the PR.

Thanks!

**Comment by BlitzErik** [comment]
Closed via #1251
